pair-programming/week-7/Factorial_Calculation_Using_Recursion.js

```diff
@@ -1,10 +1,12 @@
+// filepath: week-7/recursion-factorial/index.js
+
 /*
 Write a JavaScript function to calculate the factorial of a given number using recursion. The factorial of a non-negative integer n is the product of all positive integers less than or equal to n. It is denoted by n!.
 */
 
-function factorial() {
-  // Base case
-  // Recursive case
+function factorial(n) {
+  // Add your recursive logic here
```

Comment:
Participants are new to this concept and added explanation is paramount when providing code comments.

```suggestion
  // Add your recursive logic here (including your base case and recursive case)
```